packages/odyssey-react-mui/src/ui-shell/SideNav/SideNavItemContent.tsx

```diff
@@ -251,6 +266,7 @@ const SideNavItemContent = ({
   isSelected,
   scrollRef,
   onItemSelected,
+  translate,
```

Comment:
I think this is part of the HTMLProps in Odyssey. We should be able to `Pick` it from those so we don't have to redefine and document the type.

Reply:
yes, it is actually defined in the SidenavItem type here which picks from the HtmlProps.
https://github.com/okta/odyssey/pull/2498/files#diff-d17484d179561f56859306f5e88afebaba8d44bbf1bb2fa6474a165ce95960a3R222